docs/DeveloperGuide.md

```diff
@@ -1150,8 +1172,26 @@ testers are expected to do more *exploratory* testing.
 
 #### Help
 
-1. _{ more test cases to come …​ }_
-
+1. Test general help
```

Comment:
Same with the previous case, should describe as default state e.g all students are displayed in student list and what is the result display etc.